go/labels/common.go

```diff
@@ -8,11 +8,12 @@ import (
 )
 
 var (
-	DestinationID   = func(v string) log.Field { return log.NewStringField("destinationId", v) }
-	DestinationType = func(v string) log.Field { return log.NewStringField("destinationType", v) }
-	SourceID        = func(v string) log.Field { return log.NewStringField("sourceId", v) }
-	SourceType      = func(v string) log.Field { return log.NewStringField("sourceType", v) }
-	WorkspaceID     = func(v string) log.Field { return log.NewStringField("workspaceId", v) }
-	Namespace       = func(v string) log.Field { return log.NewStringField("namespace", v) }
-	Error           = func(v error) log.Field { return log.NewErrorField(v) }
+	DestinationID    = func(v string) log.Field { return log.NewStringField("destinationId", v) }
+	DestinationType  = func(v string) log.Field { return log.NewStringField("destinationType", v) }
+	SourceID         = func(v string) log.Field { return log.NewStringField("sourceId", v) }
+	SourceType       = func(v string) log.Field { return log.NewStringField("sourceType", v) }
+	WorkspaceID      = func(v string) log.Field { return log.NewStringField("workspaceId", v) }
+	TransformationID = func(v string) log.Field { return log.NewStringField("transformationId", v) }
```

Comment:
does this change everytime the transformation's body change?

Reply:
This is a side effect of running the linter.  It wants to align all `=` horizontally. Given the length of `TransformationID` it caused all the lines to change.

Alternatively, we can replace the `var` group with individual vars, I don't think it is worth it. It is not going to happen often

Reply:
The actual question is whether the `TransformationID` value will change every time the customer modifies the transformation script in the ui

Reply:
The transformationID remains the same but we also have a transformationVersionID which changes everytime modification happens

Reply:
thanks, so `TransformationVersionID` is the one with the highest cardinality